### PR TITLE
Adds monophobia quirk. 

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

Adds a quirk that gives you permanent monophobia.

## Why It's Good For The Game

More snowflake traits.

## Changelog
:cl:
add: new snowflake trait
/:cl:

### CODE REVIEW

**Comment by Ragolution** [comment]
Wasn't this already a thing?

**Comment by ghost** [comment]
> Wasn't this already a thing?

I remember it being a thing, but the "Configure Quirks" menu doesn't have it as an option.

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -452,3 +452,21 @@ GLOBAL_LIST_EMPTY(family_heirlooms)
...
 	lose_text = "<span class='notice'>You feel more warm-blooded.</span>"
+	
+	/datum/quirk/monophobia
+	name = "Monophobia"
+	desc = "You will become increasingly stressed when not in company of others, triggering panic reactions ranging from sickness to heart attacks."
+	value = -3 // Might change it to 4.
+	gain_text = "<span class='danger'>You feel really lonely...</span>"
+	lose_text = "<span class='notice'>You feel like you could be safe on your own.</span>"
+	medical_record_text = "Patient feels sick and distressed when not around other people, leading to potentially lethal levels of stress."
+	var/datum/brain_trauma/severe/monophobia
```
Comment (silicons) [inline code/datums/traits/negative.dm:463]:
```suggestion
	var/datum/brain_trauma/severe/monophobia/monophobia
```

File: code/datums/traits/negative.dm [hunk 2]
```diff
@@ -452,3 +452,21 @@ GLOBAL_LIST_EMPTY(family_heirlooms)
...
+
+/datum/quirk/monophobia/post_add()
+	var/mob/living/carbon/human/H = quirk_holder
+	monophobia = new
+	H.gain_trauma(monophobia, TRAUMA_RESILIENCE_ABSOLUTE)
+
+/datum/quirk/monophobia/remove()
+	var/mob/living/carbon/human/H = quirk_holder
+	H?.cure_trauma_type(monophobia, TRAUMA_RESILIENCE_ABSOLUTE)
```
Comment (silicons) [inline code/datums/traits/negative.dm]:
```suggestion
	/datum/quirk/monophobia
	name = "Monophobia"
	desc = "You will become increasingly stressed when not in company of others, triggering panic reactions ranging from sickness to heart attacks."
	value = -3 // Might change it to 4.
	gain_text = "<span class='danger'>You feel really lonely...</span>"
	lose_text = "<span class='notice'>You feel like you could be safe on your own.</span>"
	medical_record_text = "Patient feels sick and distressed when not around other people, leading to potentially lethal levels of stress."

/datum/quirk/monophobia/post_add()
	. = ..()
	var/mob/living/carbon/human/H = quirk_holder
	H.gain_trauma(/datum/brain_trauma/severe/monophobia, TRAUMA_RESILIENCE_ABSOLUTE)

/datum/quirk/monophobia/remove()
	. = ..()
	var/mob/living/carbon/human/H = quirk_holder
	H?.cure_trauma_type(/datum/brain_trauma/severe/monophobia, TRAUMA_RESILIENCE_ABSOLUTE)
```

Comment (ghost) [reply code/datums/traits/negative.dm]: Thanks.

**Comment by ghost** [comment]
Somehow me just fixing the grammar in 1 quirk made the code not compile, h e l p .

**Comment by ghost** [comment]
I had to make the grammar suck again because apparently fixing it causes the code to not compile.

**Comment by ghost** [comment]
**N I C E**

**Comment by ghost** [comment]
Uh so I tested this and it didn't actually give my character monophobia.

Edit: It works now.

**Pull request merged**